Services/Cron/README.md

```diff
@@ -0,0 +1,58 @@
+# Cron
+
+**Table of Contents**
+* [ilCronJobResult](#ilCronJobResult)
+
+## ilCronJobResult
+
+The class `ilCronJobResult` determine the status
+of a current cron job.
+
+The status are:
+* `STATUS_INVALID_CONFIGURATION`
+  This status will indicate the current configuration
+  for this cron job is not correct and MUST be
+  adjusted.
+* `STATUS_NO_ACTION`
+  This status will indicate that the cron job
+  was not executed.
+* `STATUS_OK`
+  This status indicates that the cron job
+  has been successfully finished.
+* `STATUS_CRASHED`
+  A critical failure has occurred during
+  the execution of the cron job, which led
+  to an critical error.
+  The cron job needs to be restarted by
```

Comment:
Please verifiy the described behaviour.

Reply:
I've been able to verify the described behavior. I will remove the `WIP` tag.